api/nodes/serializers.py

```diff
@@ -526,9 +528,10 @@ def to_representation(self, obj):
         type_ = getattr(meta, 'type_', None)
         assert type_ is not None, 'Must define Meta.type_'
         relation_id_field = self.fields['id']
-        attribute = relation_id_field.get_attribute(obj)
+        attribute = obj.primary_institution._id if obj.primary_institution else None
         relationship = relation_id_field.to_representation(attribute)
-
+        if relationship == 'None':
```

Comment:
This is suspect; why is the relationship_id_field serializing to the string `'None'`? Shouldn't it serialize to `None`?